app/views/signs/table/_controls.html.erb

```diff
@@ -7,4 +7,13 @@
       <%= f.button "Assign", value: :assign_topic, name: :operation, class: "button neutral" %>
     </div>
   </div>
+
+  <% if current_user&.approved? %>
```

Comment:
I don't believe this template can be rendered when a user is not signed in?

Reply:
It can't but it was causing issues for some specs that rendered the controls that didn't provide a user. I could update those specs instead

Reply:
I think that would be worth trying. AFAIK helpers like `sign_in` _should_ work